app/assets/v2/css/gitcoin.css

```diff
@@ -237,6 +237,14 @@ div.button-pink {
   max-height: 150px;
 }
 
+.mission-img-wrapper {
+    height: 250px;
+}
```

Comment:
some of the images are almost 2500px x 2500px.  mind scaling them down to be less bandwidth hogs?